src/ui/EditTask.svelte

```diff
@@ -529,49 +478,41 @@ Availability of access keys:
             <!--  Due Date  -->
             <!-- --------------------------------------------------------------------------- -->
             <label for="due" class="accesskey-first">Due</label>
-            <!-- svelte-ignore a11y-accesskey -->
-            <input
-                bind:value={editableTask.dueDate}
-                id="due"
-                type="text"
-                class="input"
-                class:tasks-modal-error={!isDueDateValid}
-                placeholder={datePlaceholder}
+            <DateEditor
+                id='due'
+                dateSymbol={dueDateSymbol}
+                bind:date={editableTask.dueDate}
+                bind:isDateValid={isDueDateValid}
+                forwardOnly={editableTask.forwardOnly}
                 accesskey={accesskey("d")}
             />
-            <code class="results">{dueDateSymbol} {@html parsedDueDate}</code>
 
             <!-- --------------------------------------------------------------------------- -->
             <!--  Scheduled Date  -->
             <!-- --------------------------------------------------------------------------- -->
             <label for="scheduled" class="accesskey-first">Scheduled</label>
-            <!-- svelte-ignore a11y-accesskey -->
-            <input
-                bind:value={editableTask.scheduledDate}
-                id="scheduled"
-                type="text"
-                class:tasks-modal-error={!isScheduledDateValid}
-                class="input"
-                placeholder={datePlaceholder}
+            <!-- removing the bind in bind:isDateValid is not changing behaviour or test results-->
```

Comment:
I just tested this, and found that removing the bind breaks the disabling of the Apply button when the date is invalid.

So I think that this comment can be removed?

Reply:
I'll remove it.